students/DennisLee/lesson10/mailroom.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python3
+
+import os
+from functools import reduce
+
+class Donor():
```

Comment:
note that you don't need parens for class def in python3

Reply:
Okay, thanks.

Do you know how long we have access to a course after a person completes it? I'd like to be able to go back and reference some of the content, if necessary.